CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ endif ()
 # -- options
 option(AZMQ_BUILD_TESTS "run unit tests" ${CMAKE_MASTER_PROJECT})
 option(BUILD_SHARED_LIBS "use the libzmq static or shared" yes)
-option(AZMQ_DEVELOPER "Add verbose warning for developers " no)
+option(AZMQ_DEVELOPER "Add verbose warning for developers " ${CMAKE_MASTER_PROJECT})
```

Comment:
Shouldn't this default to NO.   I was expecting only develops would set this to YES to get the extra warnings,  but your change will have it enabled for anyone who builds the library standalone.

Reply:
This guaranties the Developer and users see the warnings if they forget to disable it. 
(Warnings should be fixed, not suppressed, or not?)
If it is used with `FetchContents` or `CPM.cmake`, it is **off**.